### DISCUSSION TITLE
How can I enable this on lightGallery?

### DISCUSSION THREAD

**Discussion by jhbucks** [annotorious/annotorious#442] [Q&A]
Hi,

We are looking into using this with lightgalleryjs (https://www.lightgalleryjs.com/).

Anyone have any experience with integration of the two?

**Comment by rsimon** [comment]
Interesting. I don't have experience with that one in particular. But you'd probably have to listen to their `lgAfterOpen` event, and the attach Annotorious to the image element. Not sure how you'd get the handle on the image. But there are probably ways through their API, or by using a clever `querySelector` I guess.
